azdo-task/scripts/build-package.sh

```diff
@@ -54,9 +54,9 @@ echo "VERSION_MAJOR=${VERSION_MAJOR}"
 echo "VERSION_MINOR=${VERSION_MINOR}"
 echo "VERSION_PATCH=${VERSION_PATCH}"
 
-echo "::set-output name=version::$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH"
-echo "::set-output name=version_short::$VERSION_MAJOR.$VERSION_MINOR"
-echo "::set-output name=version_major::$VERSION_MAJOR"
+echo "version=$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH" >> $GITHUB_OUTPUT
```

Comment:
@stuartleeks test is failing here and I am not sure how to progress: 

Error:
```sh
  \ \   / /__ _ __ ___(_) ___  _ __  
   \ \ / / _ \ '__/ __| |/ _ \| '_ \ 
    \ V /  __/ |  \__ \ | (_) | | | |
     \_/ \___|_|  |___/_|\___/|_| |_|
                                     
  [2023-03-03T13:27:44.417Z] --set-patch-version not set
  [2023-03-03T13:27:44.417Z] 
  [2023-03-03T13:27:44.431Z] VERSION_MAJOR=0
  VERSION_MINOR=3
  VERSION_PATCH=0
  [2023-03-03T13:27:44.431Z] ./scripts/build-package.sh: line 57: $GITHUB_OUTPUT: ambiguous redirect
  [2023-03-03T13:27:44.431Z] 
  Error: Dev container exec: Command failed: bash -c echo "Starting"
  ./scripts/build-test-package.sh (exit code: undefined)
  An error occurred running a command in the container.
  Error: Command failed: bash -c echo "Starting"
  ./scripts/build-test-package.sh
  ```

Reply:
trying : ` $env:GITHUB_OUTPUT`

Reply:
Starting here: <https://github.com/community/community/discussions/36746>

Reply:
My runner is > 2.297.0

Reply:
quoting $GITHUB_OUTPUT in build-package.sh results in this error:
```sh
 \ \   / /__ _ __ ___(_) ___  _ __  
   \ \ / / _ \ '__/ __| |/ _ \| '_ \ 
    \ V /  __/ |  \__ \ | (_) | | | |
     \_/ \___|_|  |___/_|\___/|_| |_|
                                     
  [2023-03-03T13:54:26.162Z] --set-patch-version not set
  [2023-03-03T13:54:26.162Z] 
  [2023-03-03T13:54:26.180Z] VERSION_MAJOR=0
  VERSION_MINOR=3
  VERSION_PATCH=0
  ./scripts/build-package.sh: line 57: : No such file or directory
  [2023-03-03T13:54:26.181Z] 
  Error: Dev container exec: Command failed: bash -c echo "Starting"
  ./scripts/build-test-package.sh (exit code: undefined)
  An error occurred running a command in the container.
  Error: Command failed: bash -c echo "Starting"
  ./scripts/build-test-package.sh
  ```
  
  I don't have any more time this morning to continue with this, I will do my best to make some over weekend, or early next week.

Reply:
I think that there are probably a few small tweaks to your PR to get this working. Would you be open to me pushing some commits to your PR branch?

Reply:
💯, thank you!